stable/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 3.8.4
+version: 3.8.5
```

Comment:
Please bump to 3.8.6 due to CVE fix:
https://github.com/helm/charts/pull/16699

Reply:
Done 👍